src/qt/splashscreen.cpp

```diff
@@ -17,26 +17,26 @@
 #include <boost/lexical_cast.hpp>
 #include "mastercore_version.h"
 
+#include <boost/lexical_cast.hpp>
```

Comment:
It includes `<boost/lexical_cast.hpp>` two times, but uses `strprintf` (utils.h) in the end. Looks like something went wrong with the auto merge of GitHub?